test/e2e/specs/editor/various/rtl.spec.js

```diff
@@ -77,51 +98,75 @@ describe( 'RTL', () => {
 		await page.keyboard.press( 'ArrowRight' );
 		await page.keyboard.press( 'Delete' );
 
-		expect( await getEditedPostContent() ).toMatchSnapshot();
+		// Check the content.
+		const content = await editor.getEditedPostContent();
+		expect( content ).toBe(
+			`<!-- wp:paragraph -->
+<p>٠١</p>
+<!-- /wp:paragraph -->`
+		);
 	} );
 
-	it( 'should arrow navigate between blocks', async () => {
+	test( 'should arrow navigate between blocks', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
 		await page.keyboard.press( 'Enter' );
 
 		await page.keyboard.type( ARABIC_ZERO );
 		await page.keyboard.press( 'Enter' );
 		await page.keyboard.type( ARABIC_ONE );
-		await pressKeyWithModifier( 'shift', 'Enter' );
+		await page.keyboard.press( 'Shift+Enter' );
 		await page.keyboard.type( ARABIC_TWO );
 		await page.keyboard.press( 'ArrowRight' );
 		await page.keyboard.press( 'ArrowRight' );
 		await page.keyboard.press( 'ArrowRight' );
 
 		// Move to the previous block with two lines in the current block.
 		await page.keyboard.press( 'ArrowRight' );
-		await pressKeyWithModifier( 'shift', 'Enter' );
+		await pageUtils.pressKeyWithModifier( 'shift', 'Enter' );
```

Comment:
Let's update this too for consistency.

```suggestion
		await page.press( 'Shift+Enter' );
```